python/ray/tests/spark/test_data.py

```diff
@@ -0,0 +1,37 @@
+from ray.util.spark import load_spark_dataset
+from ray.util.spark.data import _convert_dbfs_path_to_local_path
+import pytest
+import os
+from pyspark.sql import SparkSession
+import sys
+import tempfile
+
+
+@pytest.fixture(scope="module", autouse=True)
+def spark():
+    spark = SparkSession.builder.getOrCreate()
+    yield spark
+    spark.stop()
```

Comment:
```suggestion
    with SparkSession.builder.getOrCreate() as spark: 
        yield spark
```

Does SparkSession work as a context manager?